chance/p5js/Popup-Circles/Index.js

```diff
@@ -0,0 +1,40 @@
+var initialOffset    = 10;
+var offset           = 25;
+var globalCircleSize = 15;
+
+var col = {
```

Comment:
Yeah. But the variable is defined globally and then overwritten locally. Never used in another scope then draw.